lib/plugin/src/index.ts

```diff
@@ -3,11 +3,12 @@ export type RCFile = {
   installs: { [id: string]: string }
   tasks: { [id: string]: string }
   commands: { [id: string]: string | string[] }
+  hooks?: { [id: string]: string | string[] }
```

Comment:
**question**: I'm a bit confused about this change – what's the difference between `hooks` and `options.hooks`? does this mean we should move `hooks` into the `options` field in our config files? why do some places still use `hooks` whilst others have deleted it in favour of `options.hooks`?

Reply:
| field | `main` | `next` | this branch |
|-|-|-|-|
| `hooks` | assigning tasks to hooks | specifying hook installation options | assigning tasks to commands (with warning) |
| `commands` | ❌  | assigning tasks to commands | assigning tasks to commands | 
| `options.hooks` | ❌ | ❌ | specifying hook installation options |

Reply:
aha, makes perfect sense, thank you!